clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -446,6 +429,31 @@ private RequestFuture<ClientResponse> sendMetadataRequest(MetadataRequest.Builde
             return client.send(node, request);
     }
 
+    /**
+     * Send Fetch Request to Kafka cluster asynchronously.
+     *
+     * This method is visible for testing.
+     *
+     * @return A future that indicates result of sent Fetch request
+     */
+    RequestFuture<ClientResponse> sendFetchRequestToNode(final FetchSessionHandler.FetchRequestData requestData,
+                                                         final Node fetchTarget) {
+        final short maxVersion = requestData.canUseTopicIds() ? ApiKeys.FETCH.latestVersion() : (short) 12;
```

Comment:
Newbie question: where is this `12` documented? Is it referring to `FetchRequestData.SCHEMA_12`/`FetchTopic.SCHEMA_12`?

Is there a way to refer to the schema number programmatically instead of via the magic number?

Reply:
Just to be clear: I know that's unrelated to your change. I'm just curious as I wasn't sure what `12` referred to, exactly 😄

Reply:
Yes, you are right. Topic IDs were first introduced in version 13 for `FetchTopic.SCHEMA` and hence, we fall back to version 12 when we don't have topic IDs to send to the server.

Reply:
I have added a comment to clarify the meaning of the number 12. A programmatic way would be to create a static method something like "getLowestVersionForTopicIDs()" but I couldn't find a good place to add it. It may also require a KIP if we add it to a class which has public APIs. Leaving it as numeral 12 with a comment for now.